README.md

```diff
@@ -21,6 +21,11 @@ Contributions are welcome.
 New contributions are subject to technical review for accuracy and editorial review for consistency.
 For an overview of what to expect from editorial review, see [Peer review guide for technical documentation](https://redhat-documentation.github.io/peer-review/#checklist).
 
+Documentation in this repository follows a modular structure described in the [Modular documentation reference guide](https://redhat-documentation.github.io/modular-docs/).
+To write new documentation, you can use [modular documentation templates](https://github.com/redhat-documentation/modular-docs/tree/main/modular-docs-manual/files).
```

Comment:
Alternatively, they can copy an existing file from `guides/common/modules/` and adapt it.

Reply:
I added this as an alternative.